### DISCUSSION TITLE
Error: Adjustment kind is empty

### DISCUSSION THREAD

**Discussion by carldes** [btschwertfeger/BiasAdjustCXX#35] [General]
Hello,

I hope you're well.

Thank you for the scripts you are making available to those working in bias correction.

I would like to use these scripts but first I tried the example. It works but when I replace with data, an error is displayed.

Can you help me solve this problem?

Thanks in advance.

Carlo
![Screen Shot 2023-12-06 at 17 07 31](https://github.com/btschwertfeger/BiasAdjustCXX/assets/147393347/32351cfd-fcd2-4c3e-b7b6-6d4a3cff550c)

**Comment by btschwertfeger** [comment]
Hello @carldes, sorry for not responding in time. I haven't seen your thread. 

The error says the adjustment kind is empty, meaning there was no value given for the `-k`/`--kind` option. 

... a long time has gone, did you fixed it or do you need further help?

**Comment by carldes** [comment]
Hello Benjamin
It's fixed !!
Thank you.
Carlo
Carlo Destouches, Ing. Civil, Msc Géosciencestel : 509 40408764 

    Le lundi 16 septembre 2024 à 13:24:58 UTC−4, Benjamin T. Schwertfeger ***@***.***> a écrit :  
 
 


Hello @carldes, sorry for not responding in time. I haven't seen your thread.

The error says the adjustment kind is empty, meaning there was no value given for the -k/--kind option.

... a long time has gone, did you fixed it or do you need further help?
